template_parser.py

```diff
@@ -330,25 +366,44 @@ def read_template_types(template_type, container_cmd, template_dir, plugins_dir)
                                 instructions['Volumes'] = {"/"+section+"-data": {}}
                                 tool_dict[template_type+"-"+section] = instructions
         else:
-            info_name = "\"all\""
+            pass
     except Exception as e:
+        print e
         pass
     return info_name, service_schedule, tool_core, tool_dict, delay_sections
 
 def main():
-    template_dir = "/var/lib/docker/data/templates/"
-    plugins_dir = "/var/lib/docker/data/plugins/"
+    """main method for template_parser. Based on the action argument given, performs the actions on the correct template files"""
+    path_dirs = PathDirs()
```

Comment:
this should be created under __main__ and then passed in as an argument to main()